lfs/scanner.go

```diff
@@ -187,6 +196,12 @@ func revListShas(refLeft, refRight string, opt ScanRefsOptions) (chan string, er
 		if refRight != "" && !z40.MatchString(refRight) {
 			refArgs = append(refArgs, refRight)
 		}
+	case ScanAll:
+		refArgs = append(refArgs, "--all")
+	case ScanDeltaToRemote:
+		refArgs = append(refArgs, refLeft, "--not", "--remotes="+opt.RemoteName)
```

Comment:
How about making this support refRight != "" as well? It's a valid use case to want to scan an existing local range with both to & from AND exclude a remote as well, even if it's not used right now.

If you'd prefer not to add that I'd prefer if the enum indicated that refRight was ignored somehow.